app/views/shared/_chat_librarian_sidebar.html.erb

```diff
@@ -1,18 +1,22 @@
 <div class='panel panel-default'>
   <div class='panel-body'>
-    <div data-library-h3lp data-jid="ic@chat.libraryh3lp.com">
+    <% if on_campus_or_su_affiliated_user? %>
+      <div data-library-h3lp data-jid="ic@chat.libraryh3lp.com">
+    <% else %>
+      <div>
```

Comment:
Do we need to close these divs or are they hanging out there?

Reply:
It gets closed down at the bottom of the file where the original one got closed.